# Messaggi del classificatore

Header header

uint8 POSE = 3
uint8 type

# If POSE
uint16 REST = 0
uint16 FIST = 1
uint16 WAVE_IN = 2
uint16 WAVE_OUT = 3
uint16 FINGERS_SPREAD = 4
uint16 DOUBLE_TAP = 5
uint16 UNKNOWN = 65535

uint16 pose